judgels-client/src/components/forms/FormSelect2/FormSelect2.jsx

```diff
@@ -11,18 +11,23 @@ export function FormSelect2({ input, className, label, meta, optionValues, optio
     return <MenuItem active={modifiers.active} key={value} onClick={handleClick} text={optionNamesMap[value]} />;
   };
 
+  const filterOption = (query, option) => {
+    return option.toLowerCase().indexOf(query.toLowerCase()) >= 0;
+  };
+
   const { onChange, ...inputProps } = input;
 
   return (
     <FormGroup className={className} labelFor={input.name} label={label} intent={getIntent(meta)}>
       <Select
         className={classNames('form-group__select', getIntentClassName(meta))}
         items={optionValues}
+        itemPredicate={filterOption}
         itemRenderer={renderOption}
         activeItem={inputProps.value}
         onItemSelect={onChange}
-        inputProps={inputProps}
-        filterable={false}
+        inputProps={{ ...inputProps, autoComplete: 'off' }}
+        filterable={true}
```

Comment:
This will make every usage of `FormSelect2` to have filter, even though it only contains a few values. E.g. in clarification filter, there's only `-` and `ASKED`; using filter here does not really make sense.

What if we make this `filterable` if there are, say more than 10 values?

Reply:
Added `isUsingFilter` if the `options.length >= 10`.